pipeline_api/src/main/java/com/splicemachine/lifecycle/DatabaseLifecycleManager.java

```diff
@@ -75,11 +75,7 @@ public static DatabaseLifecycleManager manager(){
     }
 
     public DatabaseLifecycleManager(){
-       this(Executors.newSingleThreadExecutor());
-    }
-
-    public DatabaseLifecycleManager(Executor lifecycleExecutor){
-        this.lifecycleExecutor = lifecycleExecutor;
+        this.lifecycleExecutor = Executors.newSingleThreadExecutor();
```

Comment:
Could you give the thread a name? It makes it easier to identify it in a thread dump